docs/src/running-validator/validator-reqs.md

```diff
@@ -13,16 +13,21 @@ transaction for each block the validator agrees with, which can cost up to
 
 ## Hardware Recommendations
 
+The hardware recommendations below are provided as a guide.  Operators are encouraged to do their own performance testing.
+
 - CPU
   - 12 cores / 24 threads, or more
-  - 2.8GHz, or faster
+  - 2.8GHz base clock speed, or faster
+  - SHA extensions instruction support
+    - AMD Gen 3 or newer
+    - Intel Ice Lake or newer
   - AVX2 instruction support (to use official release binaries, self-compile
     otherwise)
-  - Support for AVX512f and/or SHA-NI instructions is helpful
-  - The AMD Zen3 series is popular with the validator community
+  - Support for AVX512f is helpful
```

Comment:
It might help people if we provide an external link to what `AVX512f` is from a reputable website